es-core/src/components/ImageGridComponent.h

```diff
@@ -40,7 +40,7 @@ class ImageGridComponent : public IList<ImageGridData, T>
 	using IList<ImageGridData, T>::getTransform;
 	using IList<ImageGridData, T>::mSize;
 	using IList<ImageGridData, T>::mCursor;
-	using IList<ImageGridData, T>::Entry;
```

Comment:
Compilation error at this line.

In file included from /home/devel/temp/EmulationStation_Stable/es-app/src/views/
gamelist/BasicGameListView.cpp:1:
In file included from /home/devel/temp/EmulationStation_Stable/es-app/src/views/
gamelist/BasicGameListView.h:5:
/home/devel/temp/EmulationStation_Stable/es-app/src/components/TextListComponent
.h:33:32: error: dependent using declaration resolved to type without 'typename'
        using IList<TextListData, T>::Entry;
                                      ^